pkgs/os-specific/darwin/libtapi/default.nix

```diff
@@ -19,6 +19,10 @@ stdenv.mkDerivation rec {
   # not allowed for the stdenv.
   buildInputs = [ ncurses ];
 
+  postPatch = ''
+    patch -p3 < ${./build-tapi-command.patch}
```

Comment:
Could you open a PR at https://github.com/tpoechtrager/apple-libtapi with this change? This should help make sure this stays up to date.

Reply:
I've opened a upstream PR in https://github.com/tpoechtrager/apple-libtapi/pull/17